proposals/core.md

```diff
@@ -41,12 +41,13 @@ The interface for the new protocol would be a set of client-to-server commands,
 ```json
 {
     "id": 0,
+    "to": "browsingContext/111",

```

Comment:
I think it's debatable wether routing should be inside `params` or alongside it, and I can see the argument for alongside. It's not a input to the remote steps, and so there will probably be bits of spec and implementation that would have to treat it especially, and where it would have to be excluded from some handling if it is inside `params`.